<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Import top GWAS/QTL summary statistics — import_topSNPs • echodata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Import top GWAS/QTL summary statistics — import_topSNPs"><meta property="og:description" content="The resulting top_SNPs data.frame can be used to guide
the finemap_loci in querying and fine-mapping loci."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">echodata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.7</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/echodata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/echolocatoR_Finemapping_Portal.html">echolocatoR Fine-mapping Portal</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/RajLabMSSM/echodata/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Import top GWAS/QTL summary statistics</h1>
    <small class="dont-index">Source: <a href="https://github.com/RajLabMSSM/echodata/blob/HEAD/R/import_topSNPs.R" class="external-link"><code>R/import_topSNPs.R</code></a></small>
    <div class="hidden name"><code>import_topSNPs.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>The resulting top_SNPs data.frame can be used to guide
the finemap_loci in querying and fine-mapping loci.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="fu">import_topSNPs</span><span class="op">(</span>
  <span class="va">topSS</span>,
  show_table <span class="op">=</span> <span class="cn">TRUE</span>,
  sheet <span class="op">=</span> <span class="fl">1</span>,
  munge <span class="op">=</span> <span class="cn">TRUE</span>,
  chrom_col <span class="op">=</span> <span class="cn">NULL</span>,
  position_col <span class="op">=</span> <span class="cn">NULL</span>,
  min_POS_col <span class="op">=</span> <span class="cn">NULL</span>,
  max_POS_col <span class="op">=</span> <span class="cn">NULL</span>,
  snp_col <span class="op">=</span> <span class="cn">NULL</span>,
  pval_col <span class="op">=</span> <span class="cn">NULL</span>,
  effect_col <span class="op">=</span> <span class="cn">NULL</span>,
  locus_col <span class="op">=</span> <span class="st">"Locus"</span>,
  gene_col <span class="op">=</span> <span class="st">"Gene"</span>,
  grouping_vars <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Locus"</span><span class="op">)</span>,
  remove_variants <span class="op">=</span> <span class="cn">NULL</span>,
  verbose <span class="op">=</span> <span class="cn">TRUE</span>
<span class="op">)</span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>
    <dl><dt>topSS</dt>
<dd><p>Can be a data.frame with the top summary stats per locus.
Alternatively, you can provide a path to the stored top summary stats file.
Can be in any tabular format (e.g. excel, .tsv, .csv, etc.).
This file should have one lead GWAS/QTL hits per locus.
If there is more than one SNP per locus, the one with the
 smallest p-value (then the largest effect size) is selected as the lead SNP.
The lead SNP will be used as the center of the locus when
 constructing the locus subset files.</p></dd>
<dt>show_table</dt>
<dd><p>Create an interative data table.</p></dd>
<dt>sheet</dt>
<dd><p>If the <em>topSS</em> file is an excel sheet, 
you can specify which tab to use.
You can provide either a number to identify the tab by order,
or a string to identify the tab by name.</p></dd>
<dt>munge</dt>
<dd><p>Standardise column names.</p></dd>
<dt>chrom_col</dt>
<dd><p>Column containing chromosome (can be "chr1" or "1" format).</p></dd>
<dt>position_col</dt>
<dd><p>Column containing genomic position in units of basepairs.</p></dd>
<dt>min_POS_col</dt>
<dd><p>Column containing minimum genomic position 
(used instead of an arbitrary window size).</p></dd>
<dt>max_POS_col</dt>
<dd><p>Column containing maximum genomic position 
(used instead of an arbitrary window size).</p></dd>
<dt>snp_col</dt>
<dd><p>Column containing SNP RSIDs.</p></dd>
<dt>pval_col</dt>
<dd><p>Column containing uncorrected p-values.</p></dd>
<dt>effect_col</dt>
<dd><p>Column containing effect size (e.g. Beta, OR).</p></dd>
<dt>locus_col</dt>
<dd><p>Column containing unique locus name.</p></dd>
<dt>gene_col</dt>
<dd><p>An optional column to keep track of the causal 
gene(s) in each locus (if known).</p></dd>
<dt>grouping_vars</dt>
<dd><p>The variables that you want to group by
such that each grouping_var combination has its own index SNP.
For example, if you want one index SNP per QTL eGene - 
GWAS locus pair, you could supply:
<code>grouping_vars=c("Locus","Gene")</code>.</p></dd>
<dt>remove_variants</dt>
<dd><p>SNPs to remove from <code>topSS</code>,</p></dd>
<dt>verbose</dt>
<dd><p>Print messages.</p></dd>
</dl></div>
    <div id="value">
    <h2>Value</h2>
    <p>Munged topSNPs table.</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span class="va">topSNPs</span> <span class="op">&lt;-</span> <span class="fu">echodata</span><span class="fu">::</span><span class="fu">import_topSNPs</span><span class="op">(</span></span>
<span class="r-in">    topSS <span class="op">=</span> <span class="fu">echodata</span><span class="fu">::</span><span class="va"><a href="topSNPs_Nalls2019_raw.html">topSNPs_Nalls2019_raw</a></span>, </span>
<span class="r-in">    pval_col <span class="op">=</span> <span class="st">"P, all studies"</span>,</span>
<span class="r-in">    effect_col <span class="op">=</span> <span class="st">"Beta, all studies"</span>,</span>
<span class="r-in">    locus_col <span class="op">=</span> <span class="st">"Nearest Gene"</span>, </span>
<span class="r-in">    gene_col <span class="op">=</span> <span class="st">"QTL Nominated Gene (nearest QTL)"</span><span class="op">)</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Renaming column: P, all studies ==&gt; P</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Renaming column: Beta, all studies ==&gt; Effect</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Renaming column: Nearest Gene ==&gt; Locus</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Renaming column: QTL Nominated Gene (nearest QTL) ==&gt; Gene</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "+ Assigning gene_col and locus_col independently."</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Standardising column headers.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> First line of summary statistics file: </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> SNP	CHR	BP	Locus	Gene	Effect allele	Other allele	Effect allele frequency	Effect	SE, all studies	P	P, COJO, all studies	P, random effects, all studies	P, Conditional 23AndMe only	P, 23AndMe only	I2, all studies	Freq1, previous studies	Beta, previous studies	StdErr, previous studies	P, previous studies	I2, previous studies	Freq1, new studies	Beta, new studies	StdErr, new studies	P, new studies	I2, new studies	Passes pooled 23andMe QC	Known GWAS locus within 1MB	Failed final filtering and QC	Locus within 250KB	Locus Number	</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Returning unmapped column names without making them uppercase.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> + Mapping colnames from MungeSumstats ==&gt; echolocatoR</span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Brian Schilder, Jack Humphrey, Towfique Raj.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a>
2.0.2.9000.</p>
</div>

      </footer></div>

  


  

  </body></html>

