<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Function reference â€¢ echodata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-index">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">echodata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.7</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/echodata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/echolocatoR_Finemapping_Portal.html">echolocatoR Fine-mapping Portal</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/RajLabMSSM/echodata/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index"><colgroup><col class="alias"><col class="title"></colgroup><tbody><tr><th colspan="2">
          <h2 id="all-functions">All functions <a href="#all-functions" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="BST1.html">BST1</a></code> </p>
        </td>
        <td><p><span class="pkg">echolocatoR</span> output example: BST1 locus</p></td>
      </tr><tr><td>
          <p><code><a href="BST1_LD_matrix.html">BST1_LD_matrix</a></code> </p>
        </td>
        <td><p>LD with the lead SNP: BST1 locus</p></td>
      </tr><tr><td>
          <p><code><a href="LRRK2.html">LRRK2</a></code> </p>
        </td>
        <td><p><span class="pkg">echolocatoR</span> output example: LRRK2 locus</p></td>
      </tr><tr><td>
          <p><code><a href="MEX3C.html">MEX3C</a></code> </p>
        </td>
        <td><p><span class="pkg">echolocatoR</span> output example: MEX3C locus</p></td>
      </tr><tr><td>
          <p><code><a href="assign_lead_SNP.html">assign_lead_snp()</a></code> </p>
        </td>
        <td><p>Assign lead SNP</p></td>
      </tr><tr><td>
          <p><code><a href="check_if_empty.html">check_if_empty()</a></code> </p>
        </td>
        <td><p>Check if a file is empty</p></td>
      </tr><tr><td>
          <p><code><a href="column_dictionary.html">column_dictionary()</a></code> </p>
        </td>
        <td><p>Map column names to positions.</p></td>
      </tr><tr><td>
          <p><code><a href="coords_to_rsids.html">coords_to_rsids()</a></code> </p>
        </td>
        <td><p>Get RSIDS from SNP coordinates</p></td>
      </tr><tr><td>
          <p><code><a href="createDT.html">createDT()</a></code> </p>
        </td>
        <td><p>Interactive DT</p></td>
      </tr><tr><td>
          <p><code><a href="dt_to_granges.html">dt_to_granges()</a></code> </p>
        </td>
        <td><p>Convert data.table to GRanges object</p></td>
      </tr><tr><td>
          <p><code><a href="example_fullSS.html">example_fullSS()</a></code> </p>
        </td>
        <td><p>Store <code>fullSS_dat</code></p></td>
      </tr><tr><td>
          <p><code><a href="extract_loci.html">extract_loci()</a></code> </p>
        </td>
        <td><p>Extract loci from GWAS/QTL summary statistics</p></td>
      </tr><tr><td>
          <p><code><a href="fillNA_CS_PP.html">fillNA_CS_PP()</a></code> </p>
        </td>
        <td><p>Fill NAs in PP and CS columns</p></td>
      </tr><tr><td>
          <p><code><a href="filter_snps.html">filter_snps()</a></code> </p>
        </td>
        <td><p>Filter SNPs</p></td>
      </tr><tr><td>
          <p><code><a href="find_consensus_snps.html">find_consensus_snps()</a></code> </p>
        </td>
        <td><p>Find Consensus SNPs</p></td>
      </tr><tr><td>
          <p><code><a href="find_consensus_snps_no_polyfun.html">find_consensus_snps_no_polyfun()</a></code> </p>
        </td>
        <td><p>Find Consensus SNPs: without PolyFun</p></td>
      </tr><tr><td>
          <p><code><a href="find_top_consensus.html">find_top_consensus()</a></code> </p>
        </td>
        <td><p>Find the top Consensus SNP</p></td>
      </tr><tr><td>
          <p><code><a href="genome_wide_dir.html">genome_wide_dir</a></code> </p>
        </td>
        <td><p>Example results path for genome-wide results</p></td>
      </tr><tr><td>
          <p><code><a href="get_Kunkle2019.html">get_Kunkle2019()</a></code> </p>
        </td>
        <td><p>Example GWAS summary statistics: Kunkle2019</p></td>
      </tr><tr><td>
          <p><code><a href="get_Nalls2019.html">get_Nalls2019()</a></code> </p>
        </td>
        <td><p>Example GWAS summary statistics: Nalls2019</p></td>
      </tr><tr><td>
          <p><code><a href="get_Nalls2019_merged.html">get_Nalls2019_merged()</a></code> </p>
        </td>
        <td><p><span class="pkg">echolocatoR</span> output example: all loci</p></td>
      </tr><tr><td>
          <p><code><a href="get_header.html">get_header()</a></code> </p>
        </td>
        <td><p>Get file header</p></td>
      </tr><tr><td>
          <p><code><a href="get_sample_size.html">get_sample_size()</a></code> </p>
        </td>
        <td><p>Get sample size</p></td>
      </tr><tr><td>
          <p><code><a href="github_list_files.html">github_list_files()</a></code> </p>
        </td>
        <td><p>List files in GitHub repo</p></td>
      </tr><tr><td>
          <p><code><a href="granges_to_bed.html">granges_to_bed()</a></code> </p>
        </td>
        <td><p>Convert from GRanges to BED format</p></td>
      </tr><tr><td>
          <p><code><a href="import_topSNPs.html">import_topSNPs()</a></code> </p>
        </td>
        <td><p>Import top GWAS/QTL summary statistics</p></td>
      </tr><tr><td>
          <p><code><a href="is_ggbio.html">is_ggbio()</a></code> </p>
        </td>
        <td><p>Is an object of class ggbio</p></td>
      </tr><tr><td>
          <p><code><a href="is_ggplot.html">is_ggplot()</a></code> </p>
        </td>
        <td><p>Is an object of class ggplot</p></td>
      </tr><tr><td>
          <p><code><a href="is_granges.html">is_granges()</a></code> </p>
        </td>
        <td><p>Is an object of class GRanges</p></td>
      </tr><tr><td>
          <p><code><a href="limit_snps.html">limit_snps()</a></code> </p>
        </td>
        <td><p>Limit the number of SNPs per locus.</p></td>
      </tr><tr><td>
          <p><code><a href="locus_dir.html">locus_dir</a></code> </p>
        </td>
        <td><p>Example results path for <code>Nall2019</code> BST1 locus</p></td>
      </tr><tr><td>
          <p><code><a href="melt_finemapping_results.html">melt_finemapping_results()</a></code> </p>
        </td>
        <td><p>Melt fine-mapping results</p></td>
      </tr><tr><td>
          <p><code><a href="merge_robust.html">merge_robust()</a></code> </p>
        </td>
        <td><p>Merge robust</p></td>
      </tr><tr><td>
          <p><code><a href="mungesumstats_col_map.html">mungesumstats_col_map()</a></code> </p>
        </td>
        <td><p>Column name mappings from <span class="pkg">MungeSumstats</span> to <span class="pkg">echolocatoR</span></p></td>
      </tr><tr><td>
          <p><code><a href="mungesumstats_to_echolocatoR.html">mungesumstats_to_echolocatoR()</a></code> </p>
        </td>
        <td><p>Convert from <span class="pkg">MungeSumstats</span> to <span class="pkg">echolocatoR</span> format</p></td>
      </tr><tr><td>
          <p><code><a href="portal_metadata.html">portal_metadata()</a></code> </p>
        </td>
        <td><p>Import metadata</p></td>
      </tr><tr><td>
          <p><code><a href="portal_query.html">portal_query()</a></code> </p>
        </td>
        <td><p>Search and download fine-mapping files</p></td>
      </tr><tr><td>
          <p><code><a href="preview.html">preview()</a></code> </p>
        </td>
        <td><p>Preview formatted sum stats saved to disk</p></td>
      </tr><tr><td>
          <p><code><a href="read_bgz.html">read_bgz()</a></code> </p>
        </td>
        <td><p>Read bgz</p></td>
      </tr><tr><td>
          <p><code><a href="read_parquet.html">read_parquet()</a></code> </p>
        </td>
        <td><p>Read parquet file</p></td>
      </tr><tr><td>
          <p><code><a href="reassign_lead_snps.html">reassign_lead_snps()</a></code> </p>
        </td>
        <td><p>Reassign lead SNPs</p></td>
      </tr><tr><td>
          <p><code><a href="snp_group_colorDict.html">snp_group_colorDict()</a></code> </p>
        </td>
        <td><p>Assign colors to each SNP group</p></td>
      </tr><tr><td>
          <p><code><a href="snp_group_filters.html">snp_group_filters()</a></code> </p>
        </td>
        <td><p>Get SNP group filters</p></td>
      </tr><tr><td>
          <p><code><a href="sumstatsColHeaders.html">sumstatsColHeaders</a></code> </p>
        </td>
        <td><p><code>sumstatsColHeaders</code> from <span class="pkg">MungeSumstats</span></p></td>
      </tr><tr><td>
          <p><code><a href="topSNPs_Kunkle2019.html">topSNPs_Kunkle2019</a></code> </p>
        </td>
        <td><p>TopSS example file (processed): Kunkle2019</p></td>
      </tr><tr><td>
          <p><code><a href="topSNPs_Nalls2019.html">topSNPs_Nalls2019</a></code> </p>
        </td>
        <td><p>TopSS example file (processed): Nalls2019</p></td>
      </tr><tr><td>
          <p><code><a href="topSNPs_Nalls2019_raw.html">topSNPs_Nalls2019_raw</a></code> </p>
        </td>
        <td><p>TopSS example file: Nalls2019</p></td>
      </tr><tr><td>
          <p><code><a href="update_cols.html">update_cols()</a></code> </p>
        </td>
        <td><p>Update CS cols</p></td>
      </tr><tr><td>
          <p><code><a href="write_parquet.html">write_parquet()</a></code> </p>
        </td>
        <td><p>Write parquet file</p></td>
      </tr></tbody></table></div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Brian Schilder, Jack Humphrey, Towfique Raj.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a>
2.0.2.9000.</p>
</div>

      </footer></div>

  


  

  </body></html>

